src/components/Blocks/BlockOverview.vue

```diff
@@ -17,17 +17,25 @@ export default {
     components: { BlockCard },
     props: {
         blocks: Array
+    },
+    computed: {
+        sortedBlocks() {
+            let copy = this.blocks;
+            return copy.sort(function(a, b) {
+                return a[0].blocknumber - b[0].blocknumber;
+            });
+        }
     }
 };
 </script>
 
 <style type="scss">
 .columns {
-    column-count: 2 !important;
+    flex-wrap: wrap !important;
+    flex-grow: 1 !important;
```

Comment:
Dit kan er met oneven blokken wel voor het volgende zorgen: 
![Screenshot from 2019-03-12 23-41-24](https://user-images.githubusercontent.com/9801656/54241110-77a3bc00-4520-11e9-971a-f139cc6de60e.png)

Reply:
Ik zou zeggen columns is wel fijner.

Reply:
het nadeel met columns is dat de blokken van boven naar beneden gesorteerd zijn. dus dan krijg je weer:
1  3
2  4

Reply:
Max width zou het fixen lijkt mij

Reply:
zo beter? (dit gebruikt nog steeds decks)
![Screenshot from 2019-03-13 20-06-04](https://user-images.githubusercontent.com/6753073/54307280-8b582c80-45cb-11e9-9c03-d1502871ade3.png)